src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -472,7 +473,59 @@ const PIAFormPage = () => {
 
     return () => clearTimeout(autoSaveTimer);
   });
+  const handleBack = () => {
+    navigate(-1);
+  };
+  const handleNext = () => {
+    const tabName = pathname.split('/')[3];
 
+    switch (tabName) {
+      case 'intake':
+        navigate(pathname.replace('intake', 'collectionUseAndDisclosure'));
+        break;
```

Comment:
Intake can move to the NextSteps as well. And that is conditional too.

Reply:
yes, you are right, just check the design, the intake page does not have back and next btn, it will show the btns in next steps page

Reply:
According to the design, intake (edit page) does not have a bottom navigation. The view pages has a bottom navigation.